systemd/autoreduce-queue-processor.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
For a service file to make sense, you have to register signal handlers in the main application, `queueProcessor.py` too. This is a detail that is missing from the current deployment.

Reply:
adding the service file was moved to https://github.com/neutrons/post_processing_agent/pull/49